charts/netdata/templates/psp.yaml

```diff
@@ -19,7 +19,6 @@ spec:
   hostPorts:
   - max: 65535
     min: 0
-  privileged: true
```

Comment:
'privileged: true' is needed for [sysctlInitContainer](https://github.com/netdata/helmchart/blob/915ea3bd202aef27385638dc73fc61071cecd40b/charts/netdata/templates/daemonset.yaml#L48-L56).

I don't know how people use this option (or is it used at all). My understanding is that sysctlInitContainer is supposed to perform changes on the host system thus needs to be priviliged.

Reply:
Related article: https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/